service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -317,6 +318,19 @@ private Map<String, Object> buildModelMapForPasswordRestore(Long userId, String
         return model;
     }
 
+    @Override
+    public void sendPlaceStatusChangeNotification(PlaceStatusChangeDto dto) {
```

Comment:
I'm not sure but, maybe it makes sense before sending an email too check if the user exists in DB.
Do something like this
```
if (userRepo.existsUserByEmail(dto.getUserEmail())) {
     throw new WrongEmailException(ErrorMessage.USER_NOT_FOUND_BY_EMAIL + dto.getUserEmail());
}
```
Cause I've tried to pass an email address of not a registered user and the email was sent successfully.